python/pacemaker/_cts/audits.py

```diff
@@ -184,11 +234,26 @@ def __call__(self):
         return result
 
     def is_applicable(self):
+        """ Return True if this audit is applicable in the current test configuration. """
+
         return not self._cm.Env["DoBSC"]
 
 
 class FileAudit(ClusterAudit):
+    """ Audit the filesystem looking for various failure conditions:
+
+        * The presence of core dumps from corosync or Pacemaker daemons
```

Comment:
Completely unrelated, but this reminds me, this is rather useless these days -- when run under systemd, core dumps are in /var/lib/systemd/coredump

Reply:
We could change this to look for files in that directory.  I think it puts the program name in the core dump file name, but there's potentially a lot of globs we'd want to look for (all the daemons, tools, etc.).

Reply:
Yeah and not only that, but even core dumps from pacemaker could be old. Probably we'd need to list the directory before the lab run, then compare after. That could still get false positives if unrelated stuff crashed during that time, but that shouldn't be a big deal in practice.

Reply:
Hm, I think if we just diff the directory contents, we don't need to worry about the actual file names.  It seems unlikely that something unrelated would crash at the same time, especially given that we are completely in charge of the systems that run this stuff.

Reply:
This would be a bigger change, though.  Audits aren't really set up as before/after type things.  So perhaps what I will do is leave this as-is and create a project for fixing this.

As an aside, I kind of hate all the use of `__call__` on every class, plus making a bunch of these classes act like dicts, but not completely like dicts.  It all feels a little abusive.

Reply:
T650 and T651 opened to cover problems with audits.